app/assets/javascripts/discourse/controllers/reorder-categories.js.es6

```diff
@@ -38,29 +38,22 @@ export default Ember.Controller.extend(ModalFunctionality, Ember.Evented, {
     return anyChanged;
   }.property('categoriesBuffered.@each.hasBufferedChanges'),
 
-  saveDisabled: Ember.computed.alias('showApplyAll'),
+  @computed('showApplyAll', 'showFixIndices')
+  saveDisabled(a, b) {
```

Comment:
doh, I thought at first I needed !or, and couldn't find the computed.* for that, so... yeah, changed to computed.or.